Lib/test/test_external_inspection.py

```diff
@@ -90,10 +91,10 @@ def foo():
 
 
             expected_stack_trace = [
-                'foo',
-                'baz',
-                'bar',
-                '<module>'
+                ('foo', script_name, 15),
```

Comment:
This will be painful to maintain. I suggest importing those functions in `test_external_inspection.py` and extracting their lineno here `t.script_name.__code__.co_fileno`

Reply:
But the functions are in some random script that we execute in a subprocess. Importing it in the test suite is super painful :(